stats.go

```diff
@@ -38,6 +38,7 @@ func getInitStats() *types.Statistics {
 		Statsd:        expvar.NewMap("outputs.statsd"),
 		Dogstatsd:     expvar.NewMap("outputs.dogstatsd"),
 		Webhook:       expvar.NewMap("outputs.webhook"),
+		AzureEventHub: expvar.NewMap("outputs.eventhub"),
```

Comment:
```suggestion
		AzureEventHub: expvar.NewMap("outputs.azureeventhub"),
```
It's ugly, I agree, but clear.